frontend/src/components/ElectionForm/AddCandidate.tsx

```diff
@@ -114,11 +114,13 @@ const AddCandidate = ({ onEditCandidate, candidate, index }: CandidateProps) =>
                 </Grid>
                 :
                 <Grid item xs={2} sx={{ display: "flex", alignItems: "center" }}>
-                    <Button
-                        onClick={() => setEditCandidate(true)}
-                    >
-                        <Typography variant="h6" component="h6"> Add Details </Typography>
-                    </Button>
+                    {(process.env.FF_CANDIDATE_DETAILS || false) && <>
```

Comment:
Does " || false" do anything here? Seems like its added as some kind of protection.

Reply:
good point, I wanted it to default to false if it didn't exist, but in that case it would be undefined so the false is unnecessary

I ended up changing it to === 'true', I think that's the only way to handle booleans in environment variables